frontend/src/metabase/query_builder/components/AggregationPopover/AggregationPopover.jsx

```diff
@@ -106,26 +106,26 @@ export default class AggregationPopover extends Component {
     const aggregation = this._getAggregation();
 
     if (dimension) {
-      if (item.aggregation && item.aggregation.requiresField) {
+      if (item.aggregation?.requiresField) {
         this.commitAggregation(
           AGGREGATION.setField(item.value, dimension.mbql()),
         );
       }
     } else if (item.custom) {
       // use the existing aggregation if it's valid
-      const value = aggregation && aggregation.isValid() ? aggregation : null;
+      const value = aggregation?.isValid() ? aggregation : null;
       this.setState({
         aggregation: value,
         editingAggregation: true,
       });
-    } else if (item.aggregation && item.aggregation.requiresField) {
+    } else if (item.aggregation?.requiresField) {
       // check if this aggregation requires a field, if so then force user to pick that now, otherwise we are done
       this.setState({
         aggregation: item.value,
         choosingField: true,
       });
     } else {
-      // this includse picking a METRIC or picking an aggregation which doesn't require a field
+      // this includes picking a METRIC or picking an aggregation which doesn't require a field
```

Comment:
Nice catch.
<img src="https://media0.giphy.com/media/vNsOIY5Xp5ENhuL8Q9/giphy.gif?cid=ecf05e479acpt9r423jwodhg3i9wzvfnklczthe75k95ewzv&rid=giphy.gif&ct=g" />

Reply:
You need [more of these](https://www.google.com/search?q=sf+giants+gifs&sxsrf=ALiCzsbkHsElYghRaNOIXEu4sRjnBKmpJw:1661816768996&source=lnms&tbm=isch&sa=X&ved=2ahUKEwinmsrxne35AhV7ATQIHQR3DesQ_AUoAXoECAEQAw&biw=1855&bih=1211&dpr=2), @WiNloSt ;)